recipes/spectrum/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.6.4" %}
+{% set name = "spectrum" %}
+{% set sha256 = "7d8e3fe074b2026f61b6539c4f847d0cb1dbed41d5c6788f6100bb5884aa65ec" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/a4/c85957430db53744fd70ce726d8cd0000ee33e18da39a8132ca4e5cdb2f5/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0 
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - matplotlib
+    - numpy
+    - scipy
+    - easydev
+  run:
+    - python
+    - matplotlib
+    - numpy
+    - scipy
+    - easydev
+
+test:
+  imports:
+    - spectrum
```

Comment:
Not sure if this checks the presence of the C-extension. If not, maybe add something to ensure that it is present. I'm not familiar with the library, but maybe there is a simple import test.